dbt-spark/tests/functional/adapter/persist_docs/test_persist_docs.py

```diff
@@ -150,7 +150,17 @@ def properties(self):
         return {"schema.yml": _PROPERTIES__MODELS}
 
     def test_missing_column(self, project):
-        """spark will use our schema to verify all columns exist rather than fail silently"""
+        """
+        spark will use our schema to verify all columns exist rather than fail silently
+
+        "resolve" vs "resolved" is intentional; example error message:
+            ('42000', '[42000] [Simba][Hardy] (80) Syntax or semantic analysis error thrown in server while executing query.
+            Error message from server:
+                org.apache.hive.service.cli.HiveSQLException: Error running query: [UNRESOLVED_COLUMN.WITH_SUGGESTION]
+                org.apache.spark.sql.AnalysisException: [UNRESOLVED_COLUMN.WITH_SUGGESTION] A column, variable, or function parameter with name `name` cannot be resolve (80) (SQLExecDirectW)'
+            )
+        """
         run_dbt(["seed"])
         res = run_dbt(["run"], expect_pass=False)
-        assert "Missing field name in table" in res[0].message
+        assert "[UNRESOLVED_COLUMN.WITH_SUGGESTION]" in res[0].message
```

Comment:
looks like our databricks environments still raise the old message so we might want to check if either is true:
```suggestion
        assert any("[UNRESOLVED_COLUMN.WITH_SUGGESTION]" in res[0].message, "Missing field name in table" in res[0].message)
```

Reply:
Yeah, I noticed that. I think it's based on profile. But if you're fine with the "or" scenario, so am I.